system/include/emscripten/webaudio.h

```diff
@@ -95,19 +95,27 @@ typedef void (*EmscriptenWorkletProcessorCreatedCallback)(EMSCRIPTEN_WEBAUDIO_T
 // userData3: A custom userdata pointer to pass to the callback function. This value will be passed on to the call to the given EmscriptenWorkletProcessorCreatedCallback callback function.
 void emscripten_create_wasm_audio_worklet_processor_async(EMSCRIPTEN_WEBAUDIO_T audioContext, const WebAudioWorkletProcessorCreateOptions *options, EmscriptenWorkletProcessorCreatedCallback callback, void *userData3);
 
+// Returns the number of samples processed per channel in an AudioSampleFrame, fixed at 128 in the Web Audio API 1.0 specification, and valid for the lifetime of the audio context.
+// For this to change from the default 128, the context would need creating with a yet unexposed WebAudioWorkletProcessorCreateOptions renderSizeHint, part of the 1.1 Web Audio API.
+int emscripten_audio_context_quantum_size(EMSCRIPTEN_WEBAUDIO_T audioContext);
```

Comment:
I don't see this new API used anywhere in tests.   It looks like the test is instead reading the quantum size from the AudioSampleFrame object.   Do we need both ways to discover this?  If so can we tests this function too, if only to show how/when its useful?

Reply:
I could pass the context around in the user data, but that's a very bad usage pattern.

I'll use it to print what the quantum size is, then we at least have coverage.

Reply:
>Do we need both ways to discover this?

In more complex audio applications, there will definitely be a need to know what the quantum size is up front, before the audio graph is started running. This is e.g. for the nature of allocating internal data structures for users' own audio processing graphs, which happens before audio playback is started.

So `emscripten_audio_context_quantum_size()` is very useful for allowing that to happen.

> once the quantum is larger we'd need to query the returned hardware size before calling `emscripten_start_wasm_audio_worklet_thread_async()` with a stack size.

It is fine for users to specify stack sizes that can be well larger than any possible quantum size they choose, i.e. there's no need to squeeze oneself so tight with the stack space so that e.g. 128 quantum would fit, but 512 would not. So users might do a 64K stack size for the worklet, which should be plenty to fit all of the data, even in a huge 10.2 channels format.